src/plugins/explore/public/application/register_tabs.ts

```diff
@@ -44,7 +44,15 @@ export const registerBuiltInTabs = (tabRegistry: TabRegistryService) => {
     supportedLanguages: [EXPLORE_DEFAULT_LANGUAGE],
 
     prepareQuery: (query) => {
-      return typeof query.query === 'string' ? query.query : '';
+      const patternsField = 'message'; // TODO: pull from patterns field configured in dataset
+
+      // TODO: add quotes around take patternsField
+      // ` | patterns ${patternsField} method=brain | stats count() as count, take(${patternsField}, 1) as sample by patterns_field | sort - count | fields patterns_field, count, sample`
+
+      return typeof query.query === 'string' && query.query !== ''
```

Comment:
empty string is an active query now, since "source = *" is optional. Do we still want this empty string check?